### PR TITLE
update conversation rating flow

### PR DESCRIPTION
**Pull request by akshatdhabi**
### What do you want to achieve?
- Make changes to the conversation rating flow as it was confusing to a lot of customers 
- Comments are made optional and its shown that they are
- Rating is no more recorded in 2 parts
------
image

### CODE REVIEW

File: webplugin/css/app/mck-sidebox-1.0.css [hunk 1]
```diff
@@ -5358,24 +5358,24 @@ div#mck-rated:before{
...
     display: flex;
     align-items: center;
     justify-content: flex-start;
     border-radius: 3px;
     margin-top: 18px;
     padding: 7px;
-    border: solid 0.5px #cecbcb;
     max-width: 90%;
-    margin: 18px auto;
+	margin: 18px auto;
```
Comment (devashishmamgain) [inline webplugin/css/app/mck-sidebox-1.0.css]: code formatting

File: webplugin/css/app/mck-sidebox-1.0.css [hunk 2]
```diff
@@ -5385,9 +5385,12 @@ div#mck-rated:before{
 }
 
 .mck-send-feedback {
-	margin-left: 6px;
-    border-left: 1px solid #f6f5f5;
-	padding: 7px;
+	margin-top: 10px;
```
Comment (devashishmamgain) [inline webplugin/css/app/mck-sidebox-1.0.css]: code formatting

File: webplugin/template/mck-sidebox.html [hunk 3]
```diff
@@ -281,15 +281,9 @@ <h3>No more messages!</h3>
...
-						<textarea name="feedback" cols="34" maxlength="90" id="mck-feedback-comment" placeholder="Add a comment…" aria-label="Add a comment..." tabindex="0" rows="4"></textarea>
-						<div class="mck-send-feedback" id="mck-submit-comment" aria-disabled="false" aria-label="Send feedback" tabindex="0">
-							<span class="mck-icon-send">
-								<svg focusable="false" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 15 15">
-									<path class="km-custom-widget-fill" fill="#5553B7" fill-rule="evenodd"
-										d="M7.878 7.858l-6.032.584-1.588 5.076a.787.787 0 0 0 1.097.944l13.22-6.401a.752.752 0 0 0 0-1.353L1.364.302a.787.787 0 0 0-1.096.944l1.587 5.077 6.021.584a.475.475 0 0 1 0 .946l.002.005z">
-									</path>
-								</svg>
-							</span>
+						<textarea name="feedback" cols="34" maxlength="90" id="mck-feedback-comment" placeholder="Add a feedback (optional)" aria-label="Add a feedback" tabindex="0" rows="4"></textarea>
```
Comment (devashishmamgain) [inline webplugin/template/mck-sidebox.html]: what is the purpose of aria-label? this needs to be same as placeholder? if yes, need to add (optional)

File: webplugin/template/mck-sidebox.html [hunk 4]
```diff
@@ -281,15 +281,9 @@ <h3>No more messages!</h3>
...
+						<div class="km-custom-widget-background-color mck-send-feedback" id="mck-submit-comment" aria-disabled="false" aria-label="Send feedback" tabindex="0">
+							Submit your rating
```
Comment (devashishmamgain) [inline webplugin/template/mck-sidebox.html]: is there label defined for "Submit your rating"

Comment (akshatdhabi) [reply webplugin/template/mck-sidebox.html]: Added a label

File: webplugin/template/mck-sidebox.html (hunk 3 above)
Comment (akshatdhabi) [reply webplugin/template/mck-sidebox.html]: its used for screen readers added the (optional)

File: webplugin/css/app/mck-sidebox-1.0.css (hunk 1 above)
Comment (akshatdhabi) [reply webplugin/css/app/mck-sidebox-1.0.css]: its some bug with GitHub formatting looks fine on vscode

File: webplugin/css/app/mck-sidebox-1.0.css (hunk 2 above)
Comment (akshatdhabi) [reply webplugin/css/app/mck-sidebox-1.0.css]: same here

File: webplugin/css/app/mck-sidebox-1.0.css (hunk 1 above)
Comment (devashishmamgain) [reply webplugin/css/app/mck-sidebox-1.0.css]: see if you have tab as 8 space vs 4 space setting?

Comment (akshatdhabi) [reply webplugin/css/app/mck-sidebox-1.0.css]: Its set at 4space only

**Pull request merged**